www/js/Controller/Post_item/Postitem_web.js

```diff
@@ -15,6 +15,21 @@ elephant.controller('PostitemController_web', function($scope,$localStorage ,$io
    */
   var imageToUpload = null;
 
+  var cropContainer = document.getElementById("crop-image-container");
```

Comment:
When I change to this method code stops working. We encounter some problem in the library.